### DISCUSSION TITLE
AutoHotkey user theme for Notepad++

### DISCUSSION THREAD

**Discussion by slyfox1186** [dracula/dracula-theme#414] [General] [answered]
I created a Notepad++ user theme to coincide with Dracula for AutoHotkey (.ahk)

My user theme : https://pastebin.com/zbGwzee0

Screenshots: https://imgur.com/a/d8L2EAe

Check it out and tell me what you all think!

**Comment by zenorocha** [comment]
Hey @slyfox1186, how is this different than https://github.com/dracula/notepad-plus-plus? Is this something you could send a pull request to improve? Any thoughts @ukarim?

**Comment by slyfox1186** [comment]
I was under the impression it was customized for autohotkey's scripting language which would differ from default dracula. I can work on making a pull request. I'm just a coding enthusiast and I LOVE the dracula theme. If this isn't how this works let me know and I can take it down.

**Comment by benknoble** [comment]
@slyfox1186 I don't use notepad, but maybe it has a system for customizing syntax per language (a bit like vim lets syntax maintainers create language-specific syntax groups which end up getting themed)?

**Comment by slyfox1186** [comment]
That's exactly what it is. You create an xml file and place it in the
userDefinedthemes folder and it shows up and activates if it detects a
predefined extension such as (.ahk).

It works in harmony with your Dracula theme. I thought maybe you would be
interested in adding autohotkey as a standard language for your notepad++
theme pack.

Maybe this would be an easy template for doing so.

On Mon, Jun 15, 2020 at 3:23 PM D. Ben Knoble <notifications@github.com>
wrote:

> @slyfox1186 <https://github.com/slyfox1186> I don't use notepad, but
> maybe it has a system for customizing syntax per language (a bit like vim
> lets syntax maintainers create language-specific syntax groups which end up
> getting themed)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dracula/dracula-theme/discussions/414#discussioncomment-26369>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AI7VCU72UEAKY44MKYVC3F3RWZYJ3ANCNFSM4NRF6JUA>
> .
>


-- 
Regards,

Jeff Hollis
(678) 956-3100

**Comment by benknoble** [comment, marked as answer]
If that's the case, I would say it deserves to be a PR to the notepad repo, where it will fit in with the existing theme (it would still be part of the dracula org, if that was ever in question). That way, it would probably reach more people too.

**Comment by sobuj53** [comment]
This is just brilliant. I wish it would be a part of Dracula theme by default.

**Comment by sobuj53** [comment]
This is just brilliant. I wish it would be a part of Dracula theme by default.
